manager/config/control-plane-security/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+# Adds namespace to all resources.
+namespace: m4d-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: m4d-
+
+resources:
+- ../network-policies
```

Comment:
If it's part of default then why is it added here?

Reply:
I thought it should include it, but thinking on it again I'll remove it. tnx